backend/src/test/java/gov/cdc/usds/simplereport/validators/PatientUploadRowTest.java

```diff
@@ -0,0 +1,147 @@
+package gov.cdc.usds.simplereport.validators;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PatientUploadRowTest {
+  Map<String, String> validRowMap;
+  final List<String> requiredFields =
+      List.of(
+          "first_name",
+          "last_name",
+          "race",
+          "date_of_birth",
+          "biological_sex",
+          "ethnicity",
+          "street",
+          "state",
+          "zip_code",
+          "phone_number",
+          "employed_in_healthcare",
+          "resident_congregate_setting");
+
+  final List<String> individualFields =
+      List.of(
+          "date_of_birth",
+          "race",
+          "biological_sex",
+          "ethnicity",
+          "resident_congregate_setting",
+          "employed_in_healthcare",
+          "role",
+          "state",
+          "zip_code",
+          "country",
+          "phone_number",
+          "phone_number_type",
+          "email");
+
+  @BeforeEach
+  public void init() {
```

Comment:
Sonar is complaining about these `public` modifiers - do you mind removing them for the tests?